src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs

```diff
@@ -58,6 +58,10 @@ public sealed partial class Utf8JsonWriter : IDisposable, IAsyncDisposable
 
         private JsonWriterOptions _options; // Since JsonWriterOptions is a struct, use a field to avoid a copy for internal code.
 
+        // Cache indentation settings from JsonWriterOptions to avoid recomputating them in the hot path.
```

Comment:
Fixing I typo I introduced here:
```suggestion
        // Cache indentation settings from JsonWriterOptions to avoid recomputing them in the hot path.
```